content/cpp/concepts/math-functions/terms/isnan/isnan.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'isnan()'
+Description: 'returns non-zero value (true) if value in parameter is a NaN value; and zero (false) otherwise.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`isnan()`**  returns non-zero value (true) if value in parameter is a NaN value; and zero (false) otherwise.
+
+## Syntax
+
+```pseudo
+bool isnan(float x);  
+bool isnan(double x);  
+bool isnan(long double x);  
+bool isnan(integral x);
+```
+
+## Return Value
```

Comment:
Remove this header and include the sentence about the return value after the `pseudo` block without enclosing it in a code block.